srcv2/pages/ProjectDetailPage.js

```diff
@@ -181,7 +181,7 @@ const ProjectDetailPage = (props) => {
                             Active Contributors
                         </p>
                     </div>
-                    {sessionUser().wallet && sessionUser().wallet.invoice_macaroon &&
+                    {sessionUser().wallet && sessionUser().wallet.invoice_macaroon && sessionUser().github_handle.split('/')[3] == 'otech47' &&
```

Comment:
use the contributor ID here as an environment variable `TEMP_AUTHORIZED_SUPERUSER_ID`